docs/troubleshooting.md

```diff
@@ -75,6 +75,25 @@ Run the `heketi-cli server operations info` command and wait until the
 knowing no new operations will be accepted.
 
 
+### Checking database consistency
+
+In older versions of Heketi, database could become inconsistent due to some bugs. To check consistency of the database, the cli variant of the command is `heketi-cli db check` and the server variant of the command is `heketi db check`. These commands only need access to db. They don't collect or compare data with gluster.
```

Comment:
"database could" -> "the database could"
"to some bugs" -> "to bugs in those versions of the software"
"To check consistency" -> "To check the consistency"

Reply:
done